data-raw/minimal_sf_scenario.R

```diff
@@ -0,0 +1,43 @@
+# Aim: generate a minimal dataset corresponding to a minimal example scenario
+
+json_df = jsonlite::read_json("inst/extdata/minimal_scenario2.json", simplifyVector = TRUE)
+json_df$scenario_name
+# Try importing the data into abstreet
+json_df$people$trips[[1]]
+trip_data = dplyr::bind_rows(json_df$people$trips, .id = "person")
+trip_data
+names(trip_data)
+
+# code below could be generalised to a json_to_sf() function
+linestrings = od::odc_to_sfc(cbind(
+  trip_data$origin$Position$longitude,
+  trip_data$origin$Position$latitude,
+  trip_data$destination$Position$longitude,
+  trip_data$destination$Position$latitude
+))
+# mapview::mapview(linestrings)
```

Comment:
unsure if we import or suggest mapview, so maybe we could take this out?

Reply:
It's OK because it's not part of the package code.